apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -12,6 +12,10 @@ interface VerticalTimelineProps {
 }
 
 function VerticalTimeline({ status }: VerticalTimelineProps) {
+	const signedWaiver = status === PortalStatus.waived;
+	const attendanceConfirmed = status === PortalStatus.confirmed;
+	const sentRSVP = status === PortalStatus.attending;
```

Comment:
Suggestion: we could make the three conditions cascading so we don't have to write the triple disjunction `signedWaiver || attendanceConfirmed || sentRSVP` in two different places below.
```suggestion
	const confirmedAttendance = status === PortalStatus.confirmed || status === PortalStatus.attending;
	const signedWaiver = status === PortalStatus.waived || confirmedAttendance;
```
We would then need to revert the change to the first condition for `rsvp_component`.